packages/flutter_driver/lib/src/driver.dart

```diff
@@ -545,12 +549,38 @@ class FlutterDriver {
   ///
   /// [streams] limits the recorded timeline event streams to only the ones
   /// listed. By default, all streams are recorded.
-  Future<Timeline> traceAction(Future<dynamic> action(), { List<TimelineStream> streams: _defaultStreams }) async {
+  ///
+  /// If [retainPriorEvents] is true, retains events recorded prior to calling
+  /// [action]. Otherwise, prior events are cleared before calling [action]. By
+  /// default, prior events are cleared.
+  Future<Timeline> traceAction(
+    Future<dynamic> action(), {
+    List<TimelineStream> streams: _defaultStreams,
+    bool retainPriorEvents: false,
+  }) async {
+    if (!retainPriorEvents) {
```

Comment:
No curly braces for single-line `if` statements (have we relaxed than style requirement yet?)

Reply:
Yes, the style guide now says:

```dart
// ALSO GOOD:
if (notReady) {
  return;
}
```